ast/TreeCopying.cc

```diff
@@ -168,9 +168,11 @@ unique_ptr<Expression> Send::_deepCopy(const Expression *avoid, bool root) const
     if (!root && this == avoid) {
         throw DeepCopyError();
     }
-    return make_unique<Send>(loc, recv->_deepCopy(avoid), fun, deepCopyVec(avoid, args),
-                             block == nullptr ? nullptr
-                                              : unique_ptr<Block>(cast_tree<Block>(block->_deepCopy(avoid).release())));
+    auto send = make_unique<Send>(
+        loc, recv->_deepCopy(avoid), fun, deepCopyVec(avoid, args),
+        block == nullptr ? nullptr : unique_ptr<Block>(cast_tree<Block>(block->_deepCopy(avoid).release())));
+    send->flags = flags;
```

Comment:
Is the `send->flags = flags;` the only thing that changed here?

Reply:
Yea. I changed ast::Send to take flags in the constructor to keep consistent with the rest of the copying in this file.